edx_proctoring/backends/rest.py

```diff
@@ -142,9 +142,9 @@ def on_exam_saved(self, exam):
         """
         Called after an exam is saved.
         """
-        if 'config' not in exam and self.default_config:
+        if self.default_rules and not exam.get('rules', None):
```

Comment:
The exam should get the default rules if it explicitly defines `rules: {}` ?

Reply:
That's my assumption for the moment. There are always going to be default rules, whether they're defined on the platform level (which eventually percolate to self.default_rules) or whether they're defined on the provider's side (in which case, we don't know exactly what the defaults are). For now I think we can assume that all of the backend defaults will be defined on the platform level explicitly.

Reply:
Sorry, I was unclear. You're saying there are always default rules, but I'm talking about what is on the exam.  If the exam says `{"rules": {}}`, then it will get the defaults.  The old code wouldn't have applied defaults if the exam had `{"config": {}}`

Reply:
That's right. I think this is the better approach, to ensure that at least the platform default rules get sent to the backend. We could certainly revisit it once we have a real workflow going.